WeatherApp/DAL/Models/OpenWeather/Main.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Text.Json.Serialization;
+
+namespace DAL.Models.OpenWeather
+{
+    public class Main
```

Comment:
Not a good idea to use Main as model name

Reply:
I renamed model to "MainWeatherData".